modules/exploits/windows/imap/mercury_login.md

```diff
@@ -0,0 +1,25 @@
+Mercury/32 <= 4.01b contains an stack based buffer overflow in IMAPD LOGIN verb. Sending an specially crafted IMAP login command allows remote code execution.
+
+## Vulnerable Application
+
+        This module exploits a stack buffer overflow in Mercury/32 <= 4.01b IMAPD LOGIN verb. By sending a specially crafted login command, a buffer is corrupted, and code execution is possible. This     	       vulnerability was discovered by (mu-b at digit-labs.org).
+       
```

Comment:
Here's a link to the software:

`http://web.archive.org/web/20070119125847if_/http://ftp.usm.maine.edu/pegasus/Mercury32/m32-401b.zip`